packages/base/src/UI5Element.js

```diff
@@ -638,12 +639,14 @@ class UI5Element extends HTMLElement {
 	}
 
 	/**
-	 * Use this method in order to get a reference to element in the shadow root of a web component
+	 * Use this method in order to get a reference to an element in the shadow root of the web component or the static area item of the component
 	 * @public
+	 * @method
 	 * @param {String} refName Defines the name of the stable DOM ref
 	 */
 	getStableDomRef(refName) {
-		return this.getDomRef().querySelector(`[data-ui5-stable=${refName}]`);
+		return this.staticAreaItemDomRef.querySelector(`[data-ui5-stable=${refName}]`)
```

Comment:
I think most will be in the normal DOM ref so I would put it first (and it can return early, if found) and only then check in the static area dom ref.
Also, if staticAreaItemDomRef is undefined, it will break. Probably you should put a check for both dom refs :)